content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/delivery_three_steps.rst

```diff
@@ -28,27 +28,27 @@ three step deliveries. First, make sure the *Multi-Step Routes* option is enable
    :alt: Activate multi-step routes and storage locations in inventory settings.
 
 Next, the warehouse needs to be configured for three step deliveries. To do this, go to
-:menuselection:`Inventory --> Configuration --> Warehouses`, and click :guilabel:`Edit` to edit the
-:guilabel:`Warehouse`. Then, select :guilabel:`Pack goods, send goods in output and then deliver
-(3 steps)` for :guilabel:`Outgoing Shipments`. Then :guilabel:`Save` the changes.
+:menuselection:`Inventory --> Configuration --> Warehouses`, and click on the warehouse to edit.
+Then, select :guilabel:`Pack goods, send goods in output and then deliver (3 steps)` for
+:guilabel:`Outgoing Shipments`.
 
 .. image:: delivery_three_steps/three-step-warehouse-config.png
    :align: center
    :alt: Set outgoing shipment option to deliver in three steps.
 
 Activating three-step receipts and deliveries will create two new internal locations: a
 *Packing Zone* (WH/Packing Zone), and *Output* (WH/Output). To rename these locations, go to
-:menuselection:`Configuration --> Locations`, select the :guilabel:`Location` to change, click
-:guilabel:`Edit`, update the name, and finally, click :guilabel:`Save`.
+:menuselection:`Configuration --> Locations`, click on the :guilabel:`Location` to change, and
+update the name.
 
 Process a delivery order in three steps (pick + pack + ship)
 ============================================================
 
```

Comment:
Might be nice to add a quick 1-2 sentence intro into this section, briefly elaborating on why three-step process could be beneficial in some instances - just a suggestion.

Reply:
I put this part at the very top of the document- so the explanation of why it would be a good idea is what opens the doc. I am not sure it needs to be moved or placed again here. Or if it is not explained enough in the beginning, I can add to it as well.